CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ Table of Contents
 ### Breaking Changes
 
 * Reports before version 1.9.0 cannot be loaded anymore. Simply re-run your tests with the new recheck version to create them again.
+* When defining an ignore rule with element and attribute, this rule is now applied to the whole _subtree_, i.e. all child elements of the specified element. We found that this is more intuitive and straightforward and more often what people would expect. Also, it is more in line with the current behavior when just ignoring an element (which also ignores all of its child elements).
```

Comment:
Should we put this under bug fixes (as we did the last element sub tree)?

Reply:
Good idea. Will do.